src/js/components/molecules/icon-button/icon-button.scss

```diff
@@ -78,13 +78,13 @@
 }
 
 .icon-button--underline-large {
-    @extend %fontSecondary;
     @include font-ratio-rhythm(sec-xs);
     @include min-tablet() {
         @include font-ratio-rhythm(sec-s);
     }
 
     display: inline;
```

Comment:
as far as i know, `font-family: inherit`, is already the default, so maybe this property can be removed?